definitions/npm/@react-navigation/bottom-tabs_v5.x.x/flow_v0.104.x-/bottom-tabs_v5.x.x.js

```diff
@@ -1325,6 +1325,22 @@ declare module '@react-navigation/bottom-tabs' {
     +onPress?: (MouseEvent | PressEvent) => void,
   |};
 
+  declare export type TabBarVisibilityAnimationConfig =
+    | {|
+        +animation: 'spring',
+        +config: $Diff<
```

Comment:
From [here](https://github.com/react-navigation/react-navigation/commit/0b627304aa9fbb48de7a674ff8dfbbc495f5efe6#diff-1d4e63bf299ce11ebc7798388f17ccc4764ebe3597bf3d731aef84679247c5dfR60) it looks like `config` is optional, unless I'm reading that wrong?

Reply:
You're 100% right, I totally missed that. Thanks for the review!